<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Tobias Junghans</author>
 <class>PowerDownTimeInputDialog</class>
 <widget class="QDialog" name="PowerDownTimeInputDialog">
  <property name="windowTitle">
   <string>Power down</string>
  </property>
  <property name="windowIcon">
   <iconset resource="powercontrol.qrc">
    <normaloff>:/powercontrol/system-shutdown.png</normaloff>:/powercontrol/system-shutdown.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Please specify a timeout for powering down the selected computers:</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QSpinBox" name="minutesSpinBox">
       <property name="maximum">
        <number>1440</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>minutes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="secondsSpinBox">
       <property name="maximum">
        <number>59</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>seconds</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="powercontrol.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PowerDownTimeInputDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>384</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PowerDownTimeInputDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>384</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
